packages/near-fast-auth-signer/src/utils/multi-chain/chains/Bitcoin/Bitcoin.ts

```diff
@@ -343,33 +293,30 @@ export class Bitcoin {
    * and the derived path for the account to create a transaction. It then signs the transaction
    * using the chain signature contract and broadcasts it to the Bitcoin network.
    *
-   * @param {Object} data - The transaction data.
+   * @param {BTCTransaction} data - The transaction data.
    * @param {string} data.to - The recipient's Bitcoin address.
-   * @param {number} data.value - The amount of Bitcoin to send (in satoshis).
-   * @param {Account} account - The account object containing the user's account information.
+   * @param {number} data.value - The amount of Bitcoin to send (in BTC).
+   * @param {NearAuthentication} nearAuthentication - The object containing the user's authentication information.
    * @param {string} path - The key derivation path for the account.
    */
   async handleTransaction(
-    data: {
-      to: string;
-      value: number;
-    },
-    account: Account,
+    data: BTCTransaction,
+    nearAuthentication: NearAuthentication,
     path: string,
   ) {
-    const { address, publicKey } = await Bitcoin.deriveAddress(
-      account.accountId,
+    const satoshis = Bitcoin.toSatoshi(parseFloat(data.value));
```

Comment:
The value now it's passed as satoshis, we can remove this line. also rebase issue I think. Update the docs too